### PR TITLE
hesiod-config port

### PR DESCRIPTION
**Pull request by miriam-rittenberg**

### CODE REVIEW

File: ports/net/macathena-hesiod-config/Portfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem		1.0
+PortGroup       github 1.0
+PortGroup		config_package 1.0
+
+name			macathena-hesiod-config
+github.setup    mit-athena hesiod-config 1.6
+version			1.6
```
Comment (quentinmit) [inline ports/net/macathena-hesiod-config/Portfile:8]: You don't need version if it's the same as the tag.

**Comment by miriam-rittenberg** [comment]
I moved it into the config directory as well

**Comment by quentinmit** [comment]
Is there a metapackage this should be in?

**Comment by miriam-rittenberg** [comment]
It should be in both macathena-clients and macathena-locker, yeah. I'll add.

**Pull request merged**